src/lib/components/Image.svelte

```diff
@@ -5,14 +5,19 @@
 
 	export let image: any
 	let hasHover = false
+
+	$: imageHeight = $galleryStore.zoomLevel + 80
```

Comment:
The 80 value feels a bit too arbitrary, is it possible to use a grid layout so the height is not hardcoded, and to zoom we can just change the column width of the grid?

Reply:
@alvinosh 
The zoom feature isn't done yet. 
Did it just as starting. 
And intention of this PR is to add shadcn UI, delete modal adding.